### ISSUE TITLE
GraphRNNs for Generating Realistic Graphs

### ISSUE THREAD

**Issue by VIGS25** [deepchem/deepchem#1304]
https://arxiv.org/pdf/1802.08773.pdf

The paper linked above, talks about first converting graphs into sequences and then using RNNs on these sequences to generate graphs. They also have some new baselines, benchmarks and datasets defined and show that their method of generating graphs works much better and also improves the scalability in terms of size.

Do you reckon it would be nice to have this ported into the deepchem framework?

**Comment by rbharath** [comment]
Yes, absolutely would be interested in having this ported to DeepChem. Marking this contributions welcome.

**Comment by VIGS25** [comment]
I'll do this during my Deep Learning course next semester, if no one takes it by then :)

**Comment by peastman** [comment]
That looks like a really nice method.  It should make a great addition to DeepChem.

Be sure to include support for node and edge features as described in appendix A.6.  That will be necessary for using graphs to represent molecules.

**Comment by ShigrafS** [comment]
Hi @rbharath 
Since this issue has not already been fixed. I will be working on this.
